jekyll/_cci2/java-oom.md

```diff
@@ -13,15 +13,23 @@ Without any memory limits, the JVM pre-allocates a fraction of
 the total memory available in the system.
 CircleCI runs container based builds on large machines with lots of memory.
 Each container has a smaller memory limit than the total amount available
-on the machine. This can lead to the JVM seeing a large amount of memory
-being available to it, and trying to use more than is allocated to the
-container.
+on the machine.
 
-This pre-allocation can produce Out of Memory (OOM) errors,
-which are difficult to debug because the error messages lack detail.
+As of [June 03, 2020](https://circleci.com/changelog/#container-cgroup-limits-now-visible-inside-the-docker-executor)
+these limits are visible when using the Docker executor. This means that the recent versions of Java will correctly
+detect the number of CPUs and amount of RAM available to the job.
```

Comment:
What does the JVM pick as max heap size I’m with cgroups visible and no other flags?

Reply:
Added the deets!